test/src/math/test_vecmat.cpp

```diff
@@ -38,6 +39,7 @@ class VecmatTest : public test::FSTestFixture {
 	void SetUp() override 
 	{ 
 		test::FSTestFixture::SetUp(); 
+		// DISCUSSME: what do we do here?
```

Comment:
Tests should probably have a constant seed to make them predictable. If that value doesn't work for the new system it could be changed though.

Reply:
I updated `RandomImpl::seed()` to follow `srand()` semantics, so that if you pass 1 to `Random::seed()`, the RNG will use its default seed, which for `std::mt19937` at least is part of the C++ standard and thus should be the same across platforms.

https://www.cplusplus.com/reference/random/mersenne_twister_engine/seed/